packages/astro/src/assets/fonts/config.ts

```diff
@@ -26,9 +26,11 @@ export const requiredFamilyAttributesSchema = z.object({
 	cssVariable: z.string(),
 });
 
+const entrypointSchema = z.union([z.string(), z.instanceof(URL)]);
```

Comment:
In all our schemas, a possible `URL` always starts as a string and then is transformed. We should stick to the same: 
- base: 
  https://github.com/withastro/astro/blob/9520a23d9dbad41988a1b3a56feddb689f6b481e/packages/astro/src/core/config/schemas/relative.ts#L36-L39
- transformation
  https://github.com/withastro/astro/blob/9520a23d9dbad41988a1b3a56feddb689f6b481e/packages/astro/src/core/config/schemas/base.ts#L125-L129

Reply:
This is different tho because it's not just turning the string into a url. We need to use `createRequire().resolve` for npm packages. So I think the way it is currently is the right one

Reply:
> This is different tho because it's not just turning the string into a URL
Can you elaborate? 

> We need to use createRequire().resolve for npm packages. So I think the way it is currently is the right one
I don't understand. What's the connection between the zod type and the `createRequire`?

Reply:
Ah sorry I undestand what you mean now. To be honest I'm reluctant to put `transforms` in there since they don't apply to integrations. I think it's better if an integration is also able to specify a string, not only a URL here

Reply:
Discussed it offline